src/components/Shared/AddressForm.js

```diff
@@ -5,57 +5,47 @@ import { SystemRepository } from 'src/repositories/SystemRepository'
 import { AddressFormShell } from 'src/components/Shared/AddressFormShell'
 import { useContext, useEffect } from 'react'
 import { RequestsContext } from 'src/providers/RequestsContext'
+import toast from 'react-hot-toast'
 
-
-const AddressForm = ({
-  recordId,
-  address,
-  setAddress,
-  editMode,
-  onSubmit
-}) => {
+const AddressForm = ({ recordId, address, setAddress, editMode, onSubmit }) => {
   const { getRequest, postRequest } = useContext(RequestsContext)
 
-  function onAddressSubmit(post){
-
-    console.log(post)
-     const data = {...post ,  recordId: recordId}
-      postRequest({
-        extension: SystemRepository.Address.set,
-        record: JSON.stringify(data)
-      })
-        .then(res => {
-          data.addressId = res.recordId
-          onSubmit(data)
-        })
-
+  function onAddressSubmit(post) {
+    const data = { ...post, recordId: recordId }
+    postRequest({
+      extension: SystemRepository.Address.set,
+      record: JSON.stringify(data)
+    }).then(res => {
+      data.addressId = res.recordId
+      onSubmit(data)
+      if (res.recordId) toast.success('Record Updated Successfully')
```

Comment:
Make sure that if(res.recordId) is required.

Reply:
yes needed